dref/test_views.py

```diff
@@ -847,3 +848,27 @@ def test_dref_for_super_user(self):
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
         self.assertEqual(len(response.data['results']), 0)
+
+    def test_dref_latest_update(self):
+        dref = DrefFactory.create(
+            title='Test Title',
+            created_by=self.user,
+            modified_at=datetime(2022, 4, 18, 2, 29, 39, 793615)
+        )
+        url = f'/api/v2/dref/{dref.id}/'
+        data = {
+            'title': "New title",
+            'modified_at': datetime(2022, 2, 18, 2, 29, 39, 793615),
+        }
+
+        self.client.force_authenticate(self.user)
+        response = self.client.patch(url, data=data)
+        self.assertEqual(response.status_code, 200)
```

Comment:
This should be 400, also assert for the error response
Add a test case to verify `modified_at` is not null